stubs/attrs/attr/__init__.pyi

```diff
@@ -49,7 +49,7 @@ class Attribute(Generic[_T]):
     order: bool
     hash: Optional[bool]
     init: bool
-    converter: Optional[_ConverterType[_T]]
+    converter: Optional[Union[_ConverterType[_T],List[_ConverterType[_T]],Tuple[_ConverterType[_T]]]]
```

Comment:
`Tuple[Foo]` is a tuple that contains exactly one element, of type `Foo`. You probably meant `Tuple[Foo, ...]` instead.

Smaller nits:
- You will need to run `black`. See README.
- The `|` operator for unions is now preferred, so `Optional[Union[Foo, Bar]]` can be written as `Foo | Bar | None`.

Reply:
Yeah, black failures were cos I edited this via github. I'll do it properly next time :)

On `|` for Union -- I'm too used to targeting Py 3.6 that it's ingrained that I can't use that.

If we can use `|` (which is py 3.10 ISTR) then canwe also use `list[]` and `tuple[]`?

Reply:
Yes, `list` and `tuple` are also preferred, **but** `tuple[X, ...]` doesn't work yet. Also, nothing of this works in type aliases yet.

Reply:
Updated, I didn't change the other uses of `Optional[]` in this file as it felt to me scope creep of this PR, but I can do if you'd like.